pvlib/temperature.py

```diff
@@ -978,3 +978,397 @@ def prilliman(temp_cell, wind_speed, unit_mass=11.1, coefficients=None):
     smoothed[0] = temp_cell.values[0]
     smoothed = pd.Series(smoothed, index=temp_cell.index)
     return smoothed
+
+
+def generic_linear(poa_global, temp_air, wind_speed, u_const, du_wind,
+                   module_efficiency, absorptance):
+    """
+    Calculate cell temperature using a generic linear heat loss factor model
+    as developed in [1]_.
```

Comment:
An inelegance of sphinx: the first sentence of docstrings gets used as the function summary in the API listing, causing any included footnote/reference links to break because the associated definition is on a different page (see broken link [here](https://pvlib-python--1463.org.readthedocs.build/en/1463/reference/pv_modeling.html#pv-temperature-models)).  The workaround is to add a second sentence for the footnote like we've done in the other functions in `pvlib.temeprature`.